<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons.qrc">
    <normaloff>:/Ico/ico/sound_icon-icons.com_74449.ico</normaloff>:/Ico/ico/sound_icon-icons.com_74449.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonOpen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>32</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPrev">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>32</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPlay">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>32</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPause">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>32</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStop">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>32</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNext">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>32</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderProgress">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>731</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderVolume">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>70</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelVolume">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Volume</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>381</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDuration">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Duration</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelProgress">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Progress</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableViewPlaylist">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>751</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
